packages/multichain-network-controller/src/utils.test.ts

```diff
@@ -32,4 +35,69 @@ describe('utils', () => {
       expect(checkIfSupportedCaipChainId('eip155:1')).toBe(false);
     });
   });
+
+  describe('updateNetworkConfiguration', () => {
+    it('updates the network configuration a single EVM network', () => {
+      const network: NetworkConfiguration = {
+        chainId: '0x1',
+        name: 'Ethereum Mainnet',
+        nativeCurrency: 'ETH',
+        blockExplorerUrls: ['https://etherscan.io'],
+        defaultBlockExplorerUrlIndex: 0,
+        rpcEndpoints: [],
+        defaultRpcEndpointIndex: 0,
+      };
+      expect(updateNetworkConfiguration(network)).toStrictEqual({
+        chainId: 'eip155:1',
+        isEvm: true,
+        name: 'Ethereum Mainnet',
+        nativeCurrency: 'ETH',
+        blockExplorerUrls: ['https://etherscan.io'],
+        defaultBlockExplorerUrlIndex: 0,
+      });
+    });
+  });
+
+  describe('updateNetworkConfigurations', () => {
+    it('updates the network configurations for EVM networks', () => {
```

Comment:
Just to have emphasis on "multiple" (similar to the use of "single" for the other test)

```suggestion
    it('updates the network configurations for multiple EVM networks', () => {
```